slither/solc_parsing/declarations/contract.py

```diff
@@ -559,7 +559,13 @@ def analyze_using_for(self):
                         type_name = "*"
                     if type_name not in self._contract.using_for:
                         self._contract.using_for[type_name] = []
+                        self._contract.using_for_src[type_name] = []
+                    uf = Structure(self._contract.compilation_unit)
```

Comment:
Let's add a comment here, saying something like "We use a `Structure` declaration to record the source location for "using" declaration. For simplicity, we reused the `Structure`, instead of creating a new class `UsingFor`."